.github/workflows/continuous-integration.yml

```diff
@@ -73,12 +73,22 @@ jobs:
         php-version:
           - "7.4"
           - "8.1"
+        oracle-version:
+          - "11"
+          - "latest"
 
     services:
       oracle:
-        image: "wnameless/oracle-xe-11g-r2"
+        image: gvenzl/oracle-xe:${{ matrix.oracle-version }}
+        env:
+          ORACLE_PASSWORD: oracle
         ports:
           - "1521:1521"
+        options: >-
+          --health-cmd healthcheck.sh
+          --health-interval 20s
+          --health-timeout 10s
+          --health-retries 10
```

Comment:
Is it necessary to override the defaults? Usually, the defaults are sufficient as it has been with the previous image.

Reply:
Removed